server/athenian/api/models/metadata/github.py

```diff
@@ -39,6 +39,7 @@ class UserMixin:
 
 class RepositoryMixin:
     repository_full_name = Column(Text, nullable=False)
+    repository_node_id = Column(Text, nullable=False)
```

Comment:
This can potentially break all the unit tests :cry: New columns require corresponding data filled in `tests/test_data.sql.xz`.

Reply:
Ok, I'll have a look into it, thanks! 👍